geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/stats/RasterQueryStat.scala

```diff
@@ -40,7 +40,7 @@ object RasterQueryStatTransform extends StatTransform[RasterQueryStat] {
   val NUMBER_OF_CQ_DATA_TYPES = 6
```

Comment:
I didn't update the raster logging actually, so there's no user name added here and still 6 cols. I'll add a ticket to update this too.